src/backend/distributed/executor/multi_utility.c

```diff
@@ -1877,6 +1958,11 @@ ExecuteDistributedDDLCommand(Oid relationId, const char *ddlCommandString,
 
 	ShowNoticeIfNotUsing2PC();
 
+	if (shouldSyncMetadata)
+	{
+		SendCommandToWorkers(WORKERS_WITH_METADATA, (char *) ddlCommandString);
```

Comment:
I think we need `SET enable_ddl_propagation TO off` here to prevent it from throwing errors later.